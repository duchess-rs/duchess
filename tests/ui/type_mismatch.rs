//@compile-flags: --crate-type lib

duchess::java_package! {
    package java.lang;

    public class java.lang.Object { //~ ERROR: generic type parameter `Id { data: "bool" }` not among in-scope parameters: []
        public java.lang.Object();
        public native bool hashCode();
    }
}

fn main() {}
