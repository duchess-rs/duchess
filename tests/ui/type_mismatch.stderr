error: generic type parameter `Id { data: "bool" }` not among in-scope parameters: []
 --> $DIR/type_mismatch.rs:6:5
  |
6 |     public class java.lang.Object {
  |     ^^^^^^

error: aborting due to previous error

